zerver/openapi/python_examples.py

```diff
@@ -154,26 +155,38 @@ def get_presence(client: Client) -> None:
     # Get presence information of all the users in an organization.
     result = client.get_realm_presence()
     # {code_example|end}
+    assert result["result"] == "success"
     validate_against_openapi_schema(result, "/realm/presence", "get", "200")
 
 
 @openapi_test_function("/default_streams:post")
 def add_default_stream(client: Client) -> None:
+    result = client.add_subscriptions(
+        streams=[
+            {
+                "name": "test stream",
+                "description": "New stream for testing",
+            },
+        ],
+    )
+    stream_id = client.get_stream_id("test stream")["stream_id"]
```

Comment:
Let's not use "stream" here since we're in the process of renaming stream to channel.